client/core/trade.go

```diff
@@ -88,12 +88,12 @@ type trackedTrade struct {
 	matchMtx sync.RWMutex
 	matches  map[order.MatchID]*matchTracker
 	notify   func(Notification)
+	epochLen uint64
 }
 
 // newTrackedTrade is a constructor for a trackedTrade.
-func newTrackedTrade(dbOrder *db.MetaOrder, preImg order.Preimage, dc *dexConnection,
+func newTrackedTrade(dbOrder *db.MetaOrder, preImg order.Preimage, dc *dexConnection, mkt *Market,
```

Comment:
Unless you have plans for the `*Market` argument, can we just pass `epochLen uint64`?

Reply:
Yeah. I was on the fence with that anyway.